### ISSUE TITLE
TypeError: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by slayer** [GoogleCloudPlatform/cloud-code-vscode#147]
**Environment**:
*(Hint: "Report Kubernetes Extension Issue on Github" command will fill these out for you.)*
Extension version: Cloud Code v0.0.12 (2019-09-23)
VSCode version:
Version: 1.39.2
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Date: 2019-10-16T10:47:27.951Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 5.0.0-32-generic
OS: Ubuntu 

**Description**:
```
TypeError: Cannot read property 'map' of undefined
    at k.a (/home/vlad/.vscode-oss/extensions/googlecloudtools.cloudcode-0.0.12/dist/server/server.js:24:311676)
    at k.t (/home/vlad/.vscode-oss/extensions/googlecloudtools.cloudcode-0.0.12/dist/server/server.js:31:2899426)
    at k.emit (events.js:182:13)
    at k._readdirError (/home/vlad/.vscode-oss/extensions/googlecloudtools.cloudcode-0.0.12/dist/server/server.js:24:541773)
    at /home/vlad/.vscode-oss/extensions/googlecloudtools.cloudcode-0.0.12/dist/server/server.js:24:541070
    at FSReqWrap.oncomplete (fs.js:141:20)
[Info  - 11:06:24 AM] Connection to server got closed. Server will restart.
/home/vlad/.vscode-oss/extensions/googlecloudtools.cloudcode-0.0.12/dist/server/server.js:24
var n=function(e,t){return(n=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(e,t){e.__proto__=t}||function(e,t){for(var r in t)t.hasOwnProperty(r)&&(e[r]=t[r])})(e,t)};function i(e,t){function r(){this.constructor=e}n(e,t),e.prototype=null===t?Object.create(t):(r.prototype=t.prototype,new r)}var a=function(){return(a=Object.assign||function(e){for(var t,r=1,n=arguments.length;r<n;r++)for(var i in t=arguments[r])Object.prototype.hasOwnProperty.call(t,i)&&(e[i]=t[i]);return e}).apply(this,arguments)};function o(e,t){var r={};for(var n in e)Object.prototype.hasOwnProperty.call(e,n)&&t.indexOf(n)<0&&(r[n]=e[n]);if(null!=e&&"function"==typeof Object.getOwnPropertySymbols){var i=0;for(n=Object.getOwnPropertySymbols(e);i<n.length;i++)t.indexOf(n[i])<0&&(r[n[i]]=e[n[i]])}return r}function s(e,t,r,n){var i,a=arguments.length,o=a<3?t:null===n?n=Object.getOwnPropertyDescriptor(t,r):n;if("object"==typeof Reflect
```

**Comment by redborian** [comment]
Thanks for reporting the issue @slayer. We have fixed this bug in our next release which will be out soon.

**Comment by redborian** [comment]
This should be fixed in v0.0.13
